daemon/cmd/status.go

```diff
@@ -161,6 +161,16 @@ func (d *Daemon) getBandwidthManagerStatus() *models.BandwidthManager {
 	return s
 }
 
+func (d *Daemon) getHostRoutingStatus() *models.HostRouting {
+	s := &models.HostRouting{}
+	if option.Config.EnableHostLegacyRouting {
+		s.Mode = models.HostRoutingModeLegacy
+	} else {
+		s.Mode = models.HostRoutingModeBPF
+	}
```

Comment:
Nit: could be simplified to
```suggestion
	s := &models.HostRouting{Mode: models.HostRoutingModeBPF}
	if option.Config.EnableHostLegacyRouting {
		s.Mode = models.HostRoutingModeLegacy
	}
```